sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/join/CoGroupByKey.java

```diff
@@ -74,6 +75,16 @@
 public class CoGroupByKey<K> extends
     PTransform<KeyedPCollectionTuple<K>,
                PCollection<KV<K, CoGbkResult>>> {
+
+  /**
+   *  options to filter output results.
```

Comment:
I'd like to improve this Javadoc to include full description and specification here.